### PR TITLE
Removes Old Deprecated Stuff

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR removes old, deprecated and/or unused things from the repo. most of it old atmos stuff.

**note:** icondiffbot2 is runtiming because one of the spritesheets this PR removes is corrupt.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: code/game/machinery/pipe/construction.dm [hunk 1]
```diff
@@ -142,8 +128,6 @@ Buildable meters
 			src.pipe_type = PIPE_PASSIVE_GATE_SUPPLY
 		else if(istype(make_from, /obj/machinery/atmospherics/binary/passive_gate))
 			src.pipe_type = PIPE_PASSIVE_GATE
-		else if(istype(make_from, /obj/machinery/atmospherics/unary/heat_exchanger))
```
Comment (Generalcamo) [inline code/game/machinery/pipe/construction.dm:145]: This one might be an accidental removal?

Comment (SleepyGemmy) [reply code/game/machinery/pipe/construction.dm:145]: good catch. will fix.

**Review by Generalcamo — COMMENTED** [review]
Removal of Security Bots, Security Assault Bots, and Floor Bots is probably player facing as these did spawn in the Warehouse occasionally, and were utilized by players when they did. This probably also demands a wiki change, as otherwise we're keeping old information on the wiki regarding what Machinists can build.

**Comment by SleepyGemmy** [comment]
removal of security bots and the bigger security bot was already done on the Aurora. spawning in cargo is a holdover, including the floorbot.

**Review by alsoandanswer — DISMISSED** [review]
Leave in the simple robots such as the MULE and the ED209, as some are still able to be created by robotics. This is an undocumented player facing change, please make a seperate PR if you want to do this.

**Comment by Generalcamo** [comment]
I checked prior PRs. The simple bots were intentionally left in for robotics to build; only the round start spawns were deprecated.

**Pull request merged**